lib/Redmine/Api/AbstractApi.php

```diff
@@ -117,6 +117,36 @@ protected function sanitizeParams(array $defaults, array $params)
             array($this, 'isNotNull')
         );
     }
+
+    /**
+     * Generate a Redmine compatible URL-encoded query string
+     * 
+     * @param mixed $data
+     * @param string $parentKey
+     * @param bool $isNested
+     * @return string string a URL-encoded string.
+     */
+    function httpBuildQuery($data, $parentKey = '', $isNested = false)
```

Comment:
Yep, thats how I was originally going to implement   the fix, however I believe it's a bit unsafe.

For example, you may want to search for an item which has [123] in its title etc?